### PR TITLE
otelm folder and opds.py

### PR DESCRIPTION
**Pull request by robelgeda**
Fixes to minor bugs.
Response to #178.

### CODE REVIEW

**Comment by robelgeda** [comment]
Ready for review and merger.

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 """)
 
 import ah_bootstrap
-from setuptools import setup
+from setuptools import setup, find_packages
```
Comment (mperrin) [inline setup.py]: where does find_packages get called such that it needs to be imported here? Is it necessary for the recursive-include part to work? I have to admit I don't have a good understanding of the internals of setuptools

File: webbpsf/opds.py [hunk 2]
```diff
@@ -1248,8 +1248,7 @@ def _apply_hexikes_to_seg(self, segment, hexike_coeffs, debug=False):
 
         apmask = np.ones_like(Xc) # by construction, we're only evaluating this for the good pixels
 
-        hexikes = zernike.hexike_basis_wss(x=Xc, y=Yc, nterms=len(hexike_coeffs),
-            aperture=apmask)
+        hexikes = zernike.hexike_basis_wss(x=Xc, y=Yc, nterms=len(hexike_coeffs))
```
Comment (mperrin) [inline webbpsf/opds.py]: Hmmm, I remember adding the aperture mask parameter during the JSC tests as part of modeling the JSC unstacked image array configuration. There was a reason it was there. But something does seem to have gotten out of sync between this and poppy. I'll have to look into this on Monday. Either I'll merge this as is or figure out why the aperture mask parameter is missing in the poppy function definition. Thanks.

File: setup.py (hunk 1 above)
Comment (robelgeda) [reply setup.py]: Thats right we don't need it. I will remove that now. I found a lot of useful information/instructions on this [page](http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files).

**Pull request merged**